frb_codegen/src/generator/rust/ty_optional.rs

```diff
@@ -41,14 +32,14 @@ impl TypeRustGeneratorTrait for TypeOptionalGenerator<'_> {
             self.context.config,
         );
         let obj = match inner.wrapper_struct() {
-            // An architecture has been created so that the inner type of optional field is always IrTypeBoxed.
-            // Here, too, if we use inner.self_access("v".to_owned()), since it will go to self_access in TypeBoxedGenerator,
-            // the dereferenced value *v was returned from there, which gave the error that the external struct could not be dereferenced.
+            // An architecture has been created so that the inner type of optional field is always
+            // IrTypeBoxed. Here, too, if we use inner.self_access("v".to_owned()), since it will go to
+            // self_access in TypeBoxedGenerator, the dereferenced value *v was returned from there, which
+            // gave the error that the external struct could not be dereferenced.
             //
-            // For now, removed the parameter inner.self_access("v".to_owned())
-            // and then added a bit of a hacky method here (not used self_access function),
-            // as it only covers mirrored optional fields.
-            Some(wrapper) => format!("{}.map(|v| {}(v))", obj, wrapper,),
+            // For now, removed the parameter inner.self_access("v".to_owned()) and then added a bit of a
+            // hacky method here (not used self_access function), as it only covers mirrored optional fields.
+            Some(wrapper) => format!("{}.map(|v| {}(v))", obj, wrapper),
```

Comment:
It's been a while, but this hack made me think if it was wiser to have Optional act more like a delegate, so that a nested IrType tree wouldn't be necessary.